modules/core/05-port/types/module.go

```diff
@@ -52,6 +52,7 @@ type IBCModule interface {
 		portID,
 		channelID string,
 		counterpartyVersion string,
+		counterpartyChannelID string,
```

Comment:
could you change the ordering such that `counterpartyChannelID` comes before `counterpartyVersion`? See [proposed spec changes](https://github.com/cosmos/ibc/pull/679)?

Reply:
yes, I'll do it

Reply:
> Thanks for the contribution!
> 
> This needs a changelog entry under the API breaking section and an addition to the `docs/v2-to-v3.md` migrations docs, but I can add both in a followup pr if you feel uncertain about adding that

I can make the pr

Reply:
Can I also make pr to the spec, I really want to make contribution to the spec